pkg/cmd/log.go

```diff
@@ -39,12 +41,22 @@ func outputLog(out io.Writer) error {
 	if err != nil {
 		return err
 	}
-	for i := len(commits) - 1; i >= 0; i-- {
-		commit := commits[i]
-		fmt.Fprintf(out, "commit %s\n", commit.Id)
-		fmt.Fprint(out, "Author: Who knows <mystery@person>\n")
-		fmt.Fprint(out, "Date: Whenever\n\n")
-		fmt.Fprintf(out, "    %s\n\n", commit.Message)
+	if outputFormat == "text" {
+		for i := len(commits) - 1; i >= 0; i-- {
+			commit := commits[i]
+			fmt.Fprintf(out, "commit %s\n", commit.Id)
+			fmt.Fprint(out, "Author: Who knows <mystery@person>\n")
+			fmt.Fprint(out, "Date: Whenever\n\n")
+			fmt.Fprintf(out, "    %s\n\n", commit.Message)
+		}
+	} else if outputFormat == "json" {
+		bytes, err := json.Marshal(commits)
+		if err != nil {
+			return err
```

Comment:
I suspect that the output in this case would _not_ be json and I think it should be.